script/release-binary

```diff
@@ -18,13 +18,28 @@
 #
 set -ex
 
+function usage() {
+  echo "$0
+    -d  The bucket name to store proxy binary"
+  exit 1
+}
+
+while getopts d: arg ; do
+  case "${arg}" in
+    d) DST="${OPTARG}";;
+    *) usage;;
+  esac
+done
+
+echo "Destination bucket: $DST"
+
+# The bucket name to store proxy binary
+# DST="gs://istio-build/proxy"
```

Comment:
Can we keep 

DST="gs://istio-build/proxy"

as default for istio

if -d flag is not set, DST will use the default